### DISCUSSION TITLE
notebookcheck.net issue

### DISCUSSION THREAD

**Discussion by rraanndd00mm** [gildas-lormeau/SingleFile#1523] [Q&A]
**Describe the bug**
When i save a webpage (review article) from notebookchec.net and i click on a different benchmark type, instead of showing that, the page scrolls to the start of the page

**To Reproduce**
1. Go to "https://www.notebookcheck.net/Honor-200-Pro-review-Can-an-upper-class-smartphone-with-2D-face-recognition-outshine-its-shortcomings.868481.0.html"
2. Go to performance section and click on various types of benchmarks listed in red color font. This is how it should change to the respective benchmarks you click.
3. Now save the webpage with singlefile and try to do the same with the saved html.
4. The page automatically scrolls up to start.

**Expected behavior**
Different benchmarks should show when click on their text

**Environment**
 - OS: Win10 LTSC 2021 IOT
 - Browser: Edge
 - Version: 127

**Additional context**
i'm not sure if its javascript, but it'd be really awesome if its fixed. thanks!

**Comment by gildas-lormeau** [comment]
It's indeed related to JavaScript. The procedure to follow is described here: https://github.com/gildas-lormeau/SingleFile/blob/master/faq.md#why-dont-interactive-elements-like-folding-titles-dynamic-maps-or-carousels-work-properly-in-saved-pages.
Alternatively, you can import the settings I attached. With these settings, the interactive parts of the page are still functional once saved, see the HTML page in the zip file.

[settings-JavaScript-enabled.json](https://github.com/user-attachments/files/16457524/settings-JavaScript-enabled.json)
[Honor 200 Pro review - Can an upper-class smartphone with 2D face recognition outshine its shortcomings_ - NotebookCheck.net Reviews (8_1_2024 3_55_11 PM).html.zip](https://github.com/user-attachments/files/16457534/Honor.200.Pro.review.-.Can.an.upper-class.smartphone.with.2D.face.recognition.outshine.its.shortcomings_.-.NotebookCheck.net.Reviews.8_1_2024.3_55_11.PM.html.zip)
